pkg/tortoise/tortoise.go

```diff
@@ -737,6 +749,18 @@ func (c *Service) UpdateResourceRequest(ctx context.Context, tortoise *v1beta3.T
 		return tortoise, nil
 	}
 
+	if c.IsGlobalDisableModeEnabled() {
```

Comment:
How about adding INFO logging to know the recommendations are being skipped from applying due to the  global_disable_mode?